src/components/MultiGraph/MultiGraph.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import { Meta, StoryObj } from '@storybook/react';
+import MultiGraph from './MultiGraph';
+
+// Meta for the story
+const meta: Meta<typeof MultiGraph> = {
+  title: 'Charts/MultiGraph',
+  component: MultiGraph,
+  argTypes: {
+    lineData: { control: 'object' },
+    barData: { control: 'object' },
+    categories: { control: 'array' },
+    title: { control: 'text' },
+    showLegend: { control: 'boolean' },
+    barGap: { control: 'text' },
+  },
+};
+export default meta;
+
+// Default Story (using StoryObj)
+type Story = StoryObj<typeof MultiGraph>;
+
+export const Default: Story = {
+  args: {
+    lineData: [
```

Comment:
This will also work but you can also put it in the const and then use it directly by spreading it